src/services/activities/activities.ts

```diff
@@ -0,0 +1,27 @@
+import { $auth } from '@mtfh/common';
+import { Activity, GetActivityHistoryRequestData } from './activities.types';
+import { config } from '../config';
+
+const headers = {
+    'Content-Type': 'application/json',
+    'x-api-key': config.activitiesApiKey,
+    'Authorization': `Bearer ${$auth.getValue().token}`,
+};
+
+export const getActivityHistory = async ({
```

Comment:
useAxiosSWRInfinite here. You don't need to do the auth stuff here.

Reply:
renamed to `useActivityHistory` and using `useAxiosSWRInfinite` similarly to comments to enable pagination